ballerina/readable_character_channel.bal

```diff
@@ -16,6 +16,7 @@
 import ballerina/jballerina.java;
 
 # Represents a channel, which could be used to read characters through a given ReadableByteChannel.
+@java:ExternalDependency
```

Comment:
@heshanpadmasiri Since the `ReadableCharacterChannel` is a part of `ReadableCharacterChannel`, only marking the `ReadableChannel` should be enough right?
https://github.com/ballerina-platform/module-ballerina-io/blob/75d120d256977f0b9b2525a9ad0b94b2fa8d4d43/ballerina/channel.bal#L17

Reply:
No need to do this as all these channels are accessible from the method signatures.